reddit_service_websockets/dispatcher.py

```diff
@@ -47,11 +62,13 @@ def on_message_received(self, namespace, message):
         consumers = self.consumers.get(namespace, [])
 
         # Compress the message
-        compressed = make_compressed_frame(message, COMPRESSOR)
+        compressed = make_compressed_frame(message, COMPRESSOR) \
```

Comment:
this feels pretty awkward (expression-if split across lines). it would be more obvious just written as a proper statement-style if.